docs/maintainers/maintainers-guide.md

```diff
@@ -43,25 +43,25 @@ There are two ways to add repositories to your Repository Insight Page:
 
 ### Using Your Repository Insights Page
 
-Once you have connected your repositories, you will be redirected to your Insight Page. Here, you will see a dashboard with an overview of the repositories and the contributors who have contributed to them.
+Once you have connected your repositories, you will be redirected to your Repository Insight Page. Here, you will see a dashboard with an overview of the repositories and the contributors who have contributed to them.
```

Comment:
Can you help to double check this with me?

Because after I created a repository insights page, it redirected me to the "Personal Workspace" instead of the new created page as screen recording below.

![create new repo insights 1](https://github.com/open-sauced/docs/assets/45172775/015f47d4-d9bd-4ad1-9059-1fbb81081f9b)

Reply:
Just created a bug report for that.